tools/dbTools.py

```diff
@@ -62,7 +62,6 @@ def _check_db(self):
             CREATE TABLE IF NOT EXISTS article
             (
                 id SERIAL PRIMARY KEY,
-                title VARCHAR(255),
```

Comment:
No more titles :( ?

Reply:
Only for the Spider database, as it already has a file hash and the file url, it makes no sense to have a title (we still are returning it in our json output, as well as the url, the file hash and every other usefull attributes)

Reply:
ok so this db is the one that keep track of what you have downloaded right? makes sense